CONTRIBUTING.md

```diff
@@ -0,0 +1,34 @@
+# Special thanks for your interest in contributing, you are more than welcome!
+
+## The translation from Go to V follow those steps:
+
+1. Go2V CLI app
+   - `go2v.v`
+2. Get the Go code (path handling etc.)
+	- `transpiler/go2v.v`
+3. Generate Go AST
+	- `transpiler/get_ast.go`
+	- `transpiler/go2v.v`
```

Comment:
Not necessary to show this again.

Reply:
Okay maybe `transpiler/go2v.v` isn't useful here but IMHO `transpiler/get_ast.go` is useful

Reply:
I only meant the 2nd line.  :-)

Reply:
You want to remove "3. Generate Go AST"?

Reply:
I want to remove line 11:  "- `transpiler/go2v.v`" - that's the part that doesn't need to be repeated.  It's the second line in that section.

Reply:
I don't really see why that's an issue, `transpiler/go2v.v` is effectively used in both steps

Reply:
Only to call the next routine.  It doesn't do the AST generation.